sql/2020/04_Fonts/04_03.popular_font_hosts_by_country.sql

```diff
@@ -0,0 +1,42 @@
+#standardSQL
+#popular_fonts_host_by_country
+SELECT
+  client,
+  country,
+  url_font,
+  freq_font,
+  country_total,
+  pct_country,
+  RANK() OVER (PARTITION BY country ORDER BY pct_country DESC) AS rank
+FROM 
+(
+  SELECT
+    client,
+    country,
+    NET.HOST(url) AS url_font,
+    COUNT(0) AS freq_font,
+    SUM(COUNT(0)) OVER (PARTITION BY country) AS country_total,
+    ROUND(COUNT(0)/SUM(COUNT(0)) OVER (PARTITION BY country),2) AS pct_country,
+    ROW_NUMBER() OVER (PARTITION BY client, country ORDER BY COUNT(0) DESC) AS rank
```

Comment:
This appears to be unused

Reply:
I think `row_number()` is used to sort country by which `rank <=10` (line 42)  
Would like to hear your further thought if it is not the one you mean.

Reply:
Oh, I assumed the `WHERE rank <= 10` corresponded to the `rank` field defined on line 10, but I think you're right. It might be worth renaming one of these to be clearer.

Reply:
I like the suggestion which make them easy known.